sensor/kubernetes/complianceoperator/updater.go

```diff
@@ -165,6 +166,12 @@ func (u *updaterImpl) getComplianceOperatorInfo() *central.ComplianceOperatorInf
 		}
 	}
 
+	if err := checkWriteAccess(u.client); err != nil {
+		return &central.ComplianceOperatorInfo{
+			StatusError: err.Error(),
+		}
+	}
+
```

Comment:
What's the advantage of returning on the error vs moving this check down after setting `info` and returning the error with a populated `info` object?

Reply:
Basically is this error more or less important than `getResourceListForComplianceGroupVersion` and `checkRequiredComplianceCRDsExist` because those check if CO is actually installed?

Reply:
Good point, it isn't. We can can proceed the checks, read access is guaranteed due to Sensor's read-cluster service account. Changed it.